js/ui/status/automaticUpdates.js

```diff
@@ -387,18 +399,18 @@ var Indicator = new Lang.Class({
             return AutomaticUpdatesState.IDLE;
     },
 
-    _isNewConnection: function() {
+    _alreadySentNotification: function() {
         let connection = this._getActiveConnection();
 
         if (!connection)
-            return false;
+            return true;
```

Comment:
I'm confused... why do you return `false` here now? Following the same reasoning from yesterday, if you can't get an active connection, you can't get the userSetting and thus you can't tell whether you already sent a notification, so you should return `false` (as you were returning before), no?

Reply:
The semantics of the function changed. `true` now means "already notified; don't sent a notification". This, however, is just a defensive measure - if there is no active connection, this result is ignored.